internal/proxy/configuration.go

```diff
@@ -0,0 +1,458 @@
+package proxy
+
+import (
+	"encoding/base64"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/micro/go-micro/config"
+	"github.com/micro/go-micro/config/source/env"
+	"github.com/mitchellh/mapstructure"
+	"golang.org/x/xerrors"
+)
+
+// DefaultProxyCongig specifies all the defaults used to configure sso-proxy
+// All configuration can be set using environment variables. Below is a list of
+// configuration variables via their environment configuraiton
+//
+// SESSION_COOKIE_NAME
+// SESSION_COOKIE_SECRET
+// SESSION_COOKIE_EXPIRE
+// SESSION_COOKIE_DOMAIN
+// SESSION_COOKIE_HTTPONLY
+// SESSION_TTL_LIFETIME
+// SESSION_TTL_VALID
+// SESSION_TTL_GRACEPERIOD
+//
+// REQUESTSIGNER_KEY
+//
+// CLIENT_ID
+// CLIENT_SECRET
+//
+// SERVER_PORT
+// SERVER_TIMEOUT_SHUTDOWN
+// SERVER_TIMEOUT_READ
+// SERVER_TIMEOUT_WRITE
+//
+// METRICS_STATSD_HOST
+// METRICS_STATSD_PORT
+//
+// LOGGING_ENABLE
+// LOGGING_LEVEL
```

Comment:
comment needs to be removed. This config var was included initially, but didn't seem to actually be used anywhere.